SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -106,3 +106,6 @@ Handlebars.registerHelper 'usernameFromEmail', (email) ->
 
 Handlebars.registerHelper 'substituteTaskId', (value, taskId) ->
     value.replace('$TASK_ID', taskId)
+
+Handlebars.registerHelper 'trimS3File', (filename) ->
+    return '...' + filename.substring(filename.indexOf("service.log"));
```

Comment:
unfortunately the log name will not always be `service.log`. That is the default for the SingularityExecutor, but it could be anything for s3 logs. I might go more on the route of taking the taskId out of the name (since we are already on the task page) to make a name that is more readable and shorter without having to truncate